cpp/src/converters/nvgraph.cu

```diff
@@ -35,7 +35,8 @@ void createGraph_nvgraph(nvgraphHandle_t nvg_handle,
   CUGRAPH_EXPECTS(!((gdf_G->edgeList == nullptr) &&
                   (gdf_G->adjList == nullptr) &&
                   (gdf_G->transposedAdjList == nullptr)),
-              "Invalid API parameter");
+              "Invalid API parameter: graph data is NULL");
```

Comment:
Shouldn't we better use CHECK_GRAPH(gdf_G) here?

Reply:
I think we should. We really should not have error checks that try to roll all checks into one test

fixed